client/my-sites/plan-features-comparison/style.scss

```diff
@@ -1,13 +1,15 @@
 $plan-features-header-banner-height: 20px;
 $plan-features-sidebar-width: 272px;
 
-.plan-pill.is-in-signup {
-	right: 52%;
-	transform: translate(50%);
-	border-radius: 4px;
-	background-color: var(--studio-green-5);
-	color: var(--studio-gray-80);
-	font-size: $font-body-small;
+.plan-features-comparison__content {
```

Comment:
I broke this out to another PR: https://github.com/Automattic/wp-calypso/pull/68184 since it's a quick fix to review and merge.